activejob/lib/active_job/logging.rb

```diff
@@ -4,17 +4,23 @@
 require "active_support/logger"
 
 module ActiveJob
-  module Logging # :nodoc:
+  module Logging
     extend ActiveSupport::Concern
 
     included do
+      ##
+      # :singleton_method:
+      #
+      # Accepts a logger conforming to the interface of Log4r which can be
```

Comment:
Should we say that the interface needs to conform to _either_ a Log4r logger, or the Ruby logger class? Most of the docs around configuring loggers seem to mention both as being acceptable.

Reply:
Interesting, I basically just copied this from `ActiveRecord::Core`, where did you see Ruby logger mentioned?

Reply:
Pretty much all of the `config.<>.logger` options, e.g. https://guides.rubyonrails.org/configuring.html#config-assets-logger, https://guides.rubyonrails.org/configuring.html#config-active-record-logger, https://guides.rubyonrails.org/configuring.html#config-action-controller-logger

Reply:
Ah, nice, I'll update `ActiveRecord::Core` and this PR as well then